### PR TITLE
Blog branch

### PR DESCRIPTION
**Pull request by dsanchez185**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [ ] Effective use of box model: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 15 points

**Checkpoint 2**
- [ ] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [ ] Effective use of CSS Grid: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Site is hosted on github: 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 15 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 15 points

### CODE REVIEW

File: 02week/blog/css/style.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+/* unvisited link */
+
+a:link {
+  color: #373737;
+}
+
+/* visited link */
+
+a:visited {
+  color: #373737;
```
Comment (situlillian) [inline 02week/blog/css/style.css:17]: these appear to be the same value, so you could try something like

```
a:link, a:visited {
  color: #373737
}
```

File: 02week/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      Spontaneity is the best kind of adventure.
+    </p>
+  </div>
+
+  <div class="row">
+    <div class="leftcolumn">
+      <div class="card">
+        <h2>Splash into Summer</h2>
+        <h5><a href="http://atpearl.com/">Splash Pad at the Pearl Brewery</a>, July 22, 2018</h5>
+        <img class="Body_Images" alt="The Pearl Splash Pad" src="http://3snpdc2ba9m5uwuk62n8cs84-wpengine.netdna-ssl.com/wp-content/uploads/2017/06/BonnieArbittier_cellars_pearl_farmers_market_rooftop_interior_hotel_emma-2.jpg" width="580" height="340">
```
Comment (situlillian) [inline 02week/blog/index.html:24]: as best practice, try not to start any class names, variables, or other with capital letters. those are reserved for special cases, which you'll learn more about later.

an alternate class name could be: `bodyImage` or `body_image`

**Review by situlillian — APPROVED** [review]
Great job, I love the layout and styling. Go ahead and merge!

**Comment by dsanchez185** [comment]
Hi there,
Thank you so much for the knowledge on the caps as well as using a comma
for two styles!

I was told not to merge...should I click on "Merge Pull Request"?

Diana M. Sanchez
E: dsanchez185@gmail.com
P: 210-412-4915

**Comment by situlillian** [comment]
Yes, go ahead and merge it. I don't have write access to your repo so I cannot merge :(

**Comment by situlillian** [comment]
I still do not have write access

**Comment by situlillian** [comment]
should add us like the below:

screen shot 2018-07-25 at 8 21 05 pm

**Comment by situlillian** [comment]
see how it says 'write access' ?

**Comment by dsanchez185** [comment]
Yes, I have you guys...I believe:



Diana M. Sanchez
E: dsanchez185@gmail.com
P: 210-412-4915

**Pull request merged**